receipt_for_lovely_loveseats/checkout.py

```diff
@@ -0,0 +1,41 @@
+class Checkout:
+  def __init__(self, products = {}, items = []):
+    self.products = products
+    self.items = items
+
+  def add_to_checkout(self, product_key, count):
+    item = self.__find_item(product_key)
+
+    if item is None:
+      self.items.append({ 'product_key': product_key, 'count': count })
```

Comment:
Consider extracting the shape of the item object into a private method and use the method to generate the objects.

Creating the item object only appears twice in the code at the moment, but taking some proactive DRYing for the shaping is fine.

Reply:
Looks like a named tuple or data class may be a more appropriate way to structure this type of object.